synapse/config/workers.py

```diff
@@ -132,6 +132,19 @@ def read_config(self, config, **kwargs):
 
         self.events_shard_config = ShardedWorkerHandlingConfig(self.writers.events)
 
+        # Whether this worker should run background tasks or not.
+        #
+        # As a note for developers, the background tasks guarded by this should
+        # be able to run on only a single instance (meaning that they don't
+        # depend on any in-memory state of a particular worker).
+        #
+        # No effort is made to ensure only a single instance of these tasks is
+        # running.
+        instance = config.get("run_background_tasks_on") or "master"
```

Comment:
Can we rename this local variable `bg_tasks_instance` or something, just because my brain got hung up on it a bit